### PR TITLE
FIXES #1089 - Correct PFD attitude indicator onground/inflight

### LINKED ISSUE #1089: [BUG] Incorrect PFD attitude indicator when on the ground
**Mod Version**

Latest

**Describe the bug**

When on the ground, the nose-down pitch degree lines are visible on the PFD attitude indicator. They should be missing and replaced with the plain brown color.

**To Reproduce**

1. Start APU or connect ground power.
2. Notice PFD markings.
3.

**Expected behavior**

The brown portion (negative pitch) of the attitude indicator on the PFD should be plain, with no white degree markings when on the ground.

**Actual behavior**

Attitude indicator looks the same as it does in the air, with pitch degree markings on both the nose-up and nose-down sides.

**References**

Here is the FFA320 in XP11 on the ground, notice the bottom half of the attitude indicator (artificial horizon) has no markings on the ground:
A320 - 2020-10-04 10 47 02

Here is the FBW n32nx on the ground, notice the bottom half of the attitude indicator (artificial horizon) has white markings on it on the ground:
Screenshot (74)

**Additional context**

Was this working before/when did the issue start occurring?

Is this a problem in the vanilla unmodded game?

Discord username (if different from GitHub):

### PR DESCRIPTION
**Pull request by MMontalto**
Fixes #1089 

## Summary of Changes
- Fixed PFD operation on the ground and in flight

## Screenshots (if necessary)
image
image
image

## References
https://youtu.be/ECV3IBkzZLc
https://youtu.be/TEi-WX08ov8
https://user-images.githubusercontent.com/70450099/95107937-8fe1ba00-0708-11eb-967d-0b3dfbccbbd5.png

## Additional context

Discord username (if different from GitHub): PiCcy

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build.py will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, slick on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

**Review by Carter633 — APPROVED** [review]
Name: Carter633 #1436
Date of testing: Current date 08/10/2020
Version of the sim: 1.9.3.0
PR Tested: #1189

Testing technique: Full flight from Cold and Dark
Steps you've made to test the Pull Request:
1. Spawned at Dublin, VATSIM IFR flight to Heathrow - cold & dark start on stand
2. Checked PFD attitude indicators on ground, on stand and during taxi and take off
3. Checked throughout flight
4. Checked after touchdown and during taxi back to stand
5. Also checked after shut down on stand

Overall Rating: Good
Conclusions of the testing: Works as described, no issues found

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -28,22 +28,9 @@
...
-1. [ECAM] Fix ECAM wheel page being selected before all gears are downlocked - @lukecologne (luke)
-1. [MISC] Added standby instrument brightness adjustment and attitude reset - @2hwk (2Cas#1022 on discord)
-1. [MISC] Added standby instrument bugs page and corrected standby instrument font and behaviour on power loss - @2hwk (2Cas#1022 on discord)
-1. [PFD] Added the ability to display metric altitudes - @lousybyte (lousybyte)
-1. [ECAM] Added messages related to anti-skid / N.W. steering being off - @Benjozork (Benjamin Dupont)
-1. [CDU] Updated HOLD Page to honeywell spec, PLAN page updated to show holds - @sepiroth887 (sepiroth887)
-1. [FCU] Fixed incrementing the QNH while in hPa mode - @lousybyte (lousybyte)
-1. [CDU] Follow Green Dot speed on approach with flaps clean - @veikkos (Veikko Soininen)
-1. [ND] Fixed cyan heading bug not showing on ND in VOR and LS modes - @AdenFlorian (David Valachovic)
+1. [PFD] Correct PFD attitude indicator onground/inflight @MMontalto (PiCcy)
```
Comment (Venorcis) [inline .github/CHANGELOG.md:33]: changelog merge did not go correctly

Comment (MMontalto) [reply .github/CHANGELOG.md:47]: I reverted to the master and added my own comment, correct?
Thanks, I'm sorry

**Comment by Benjozork** [comment]
@Carter633 you should only test when the "Ready to test" label has been applied, as we might request changes when it isn't.

**Comment by Carter633** [comment]
Tested because it appeared in the QA chat thanks to a bot that's been enabled in that chat? Perhaps it should be switched off to prevent people trying to be helpful testing when the bot puts it into the chat on discord.On 8 Oct 2020 14:55, Benjamin Dupont <notifications@github.com> wrote:
@Carter633 you should only test when the "Ready to test" label has been applied, as we might request changes when it isn't.

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/AttitudeIndicator.js [hunk 2]
```diff
@@ -432,10 +445,25 @@ class Jet_PFD_AttitudeIndicator extends HTMLElement {
...
+            if ((this.horizonAngle * this.horizonAngleFactor) < (this.horizonHeight / 2)) {
+                this.horizon_bottom.setAttribute("transform", "rotate(" + this.bankAngle + ", 0, 0) translate(0," + (this.horizonAngle * this.horizonAngleFactor) + ")");
+            } else {
+                this.horizon_bottom.setAttribute("transform", "rotate(" + this.bankAngle + ", 0, 0) translate(0," + this.horizonHeight / 2 + ")");
+            }
         }
         if (this.attitude_pitch) {
             this.attitude_pitch.setAttribute("transform", "translate(0," + (this.pitchAngle * this.pitchAngleFactor) + ")");
+
+            var hPitchContainer;
```
Comment (lukecologne) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/AttitudeIndicator.js:457]: Use let instead of var

**Comment by lukecologne** [comment]
Can you lower the border (don't know how to call it) when it's fully down? In RL it looks like it's closer to the radio altimeter
Sim:
grafik
RL:
grafik

**Comment by MMontalto** [comment]
Yes of course, thanks for the advice, I'm still learning! :S

**Comment by Benjozork** [comment]
> Tested because it appeared in the QA chat thanks to a bot that's been enabled in that chat? Perhaps it should be switched off to prevent people trying to be helpful testing when the bot puts it into the chat on discord.On 8 Oct 2020 14:55, Benjamin Dupont <notifications@github.com> wrote:
> @Carter633 you should only test when the "Ready to test" label has been applied, as we might request changes when it isn't.
> 
> —You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

I'll bring this up with the team, it's indeed confusing.

**Comment by MMontalto** [comment]
@lukecologne 

image
image

What do you think about it?
I also added, watching videos, a small increase in the font size of the "radio altitude" when it changes color to orange below 400 feet.
From the screens maybe you don't see very well...
Thanks again!

**Comment by lukecologne** [comment]
Huh, you're right, the font size actually increases! Good catch, I didn't even notice that when watching the video

**Comment by AdenFlorian** [comment]
looks great! thanks!

**Pull request merged**